### ISSUE TITLE
What is the correspondence between image concepts and 27 categories in the training and test sets?

### ISSUE THREAD

**Issue by busiqiao** [gifale95/eeg_encoding#6]
Thank you very much for the open access dataset, but now I'm having some issues:
I noticed that you split 1854 image concepts into a training set and a test set. The ratio of image concepts in the training and test sets is the same in the 27 high-level categories. I would like to know what the high-level categories are for each image concept in the training and test sets. I looked at the relevant documents and didn't find their correspondence. So I'd like to know how you guys make sure that the premium category is the same proportion. This is because the high-level categories corresponding to image concepts in the THINGS dataset are missing in both Bottom-up Category (Human Raters) and Top-down Category (WordNet) classification methods.

**Comment by gifale95** [comment]
Hi, I split the images based on the 27 high-level categories provided by the [THINGS database](https://osf.io/jum2f/files/osfstorage). However, I did this many years ago, and cannot find the code I used for it, so unfortunately I cannot answer your question.
